ctrlf.el

```diff
@@ -753,8 +762,9 @@ And self-destruct this hook."
 
 ;;;; Main entry point
 
-(defun ctrlf--start ()
-  "Start CTRLF session assuming config vars are set up already."
+(defun ctrlf--start (&optional arg pos)
```

Comment:
If you think of better variable names, let me know.

Reply:
What about INITIAL-CONTENTS and POSITION, following `read-from-minibuffer`?

Reply:
_Much better._ Thanks!